tox.ini

```diff
@@ -30,4 +31,4 @@ commands =
 [testenv:flake8]
 basepython = {[test]basepython}
 deps = flake8
-commands = flake8 .
+commands = flake8 setup.py pypom
```

Comment:
If that's the case you could add a `[flake8]` section with `excludes = .tox` or similar.